modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -341,6 +341,7 @@ private PartialNode startPartialNode(
         SchemaManager schemaManager = new SchemaManager(registry, tablesConfig, metaStorageMgr);
 
         DistributionZoneManager distributionZoneManager = new DistributionZoneManager(
+                null,
```

Comment:
I am sure that it would be more correct to inject `registry` here - as it is done for `TableManager`.

Reply:
I have added a registry